.circleci/config.yml

```diff
@@ -25,11 +36,79 @@ jobs:
       - store_artifacts:
           path: build/reports/tests
 
+  build:
+    executor: builder
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - gradle-{{ checksum "build.gradle.kts" }}
+            - gradle-
+      - run: ./gradlew build
+      - run:
+          name: Record the Application Version
+          command: |
+            jar xvf build/libs/court-hearing-event-*.jar BOOT-INF/classes/META-INF/build-info.properties
+            APP_VERSION=$(grep 'build.version=' BOOT-INF/classes/META-INF/build-info.properties | awk -F= '{print $2}')
+            rm BOOT-INF/classes/META-INF/build-info.properties
+            # make up a good version name
+            SHORT_HASH=$(git rev-parse --short HEAD)
+            TAG=${APP_VERSION}.${SHORT_HASH}
+            if [[ "${CIRCLE_BRANCH}" == "main" ]]; then
+              TAG=${APP_VERSION}
+            fi
+            echo $TAG > ./app_version.txt
+      - save_cache:
+          paths:
+            - ~/.gradle
+          key: gradle-{{ checksum "build.gradle.kts" }}
+      - store_test_results:
+          path: build/test-results
+      - store_artifacts:
+          path: build/reports/tests
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build/libs
+            - build.gradle.kts
+            - app_version.txt
+
+  deploy_dev:
```

Comment:
Again these jobs double up the `hmpps/deploy_env` jobs lower down in the workflow

Reply:
Removed